jdaviz/core/loaders/resolvers/url/url.py

```diff
@@ -0,0 +1,34 @@
+from traitlets import Bool, Unicode, observe
+from urllib.parse import urlparse
+
+from jdaviz.core.registries import loader_resolver_registry
+from jdaviz.core.loaders.resolvers import BaseResolver
+from jdaviz.core.user_api import LoaderUserApi
+from jdaviz.utils import download_uri_to_path
+
+
+@loader_resolver_registry('url')
+class URLResolver(BaseResolver):
+    template_file = __file__, "url.vue"
+
+    url = Unicode("").tag(sync=True)
+    cache = Bool(True).tag(sync=True)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    @property
+    def user_api(self):
+        return LoaderUserApi(self, expose=['url', 'cache'])
+
+    @property
+    def is_valid(self):
+        return urlparse(self.url).scheme in ['http', 'https']
```

Comment:
Do we want to throw in FTP here? Maybe not?

Reply:
We can consider adding that later... I don't want to overcomplicate things now.  Does `download_uri_to_path` handle `ftp` or would we need separate logic?

Reply:
I don't remember on the top of my head and I am doing other stuff now. FTP isn't critical for now but I think astropy does support it.